f3/chain.go

```diff
@@ -38,6 +38,11 @@ func (t TipSet) IsZero() bool {
 	return t.Epoch == 0 && t.CID.IsZero()
 }
 
+// Compares tipsets for equality.
+func (t *TipSet) Eq(other *TipSet) bool {
+	return t.Epoch == other.Epoch && t.CID == other.CID
+}
+
```

Comment:
This might be stray from a rebase. I don't think we need it, I removed it in #79. We can just use `==`.